ha/core.go

```diff
@@ -395,6 +379,9 @@ func (n *Node) receiveAdvertisements() {
 				log.Fatalf("receiveAdvertisements: Unable to write to errChannel. Error was: %v", err)
 			}
 		} else if advert != nil {
+			if advert.VersionType != vrrpVersionType || advert.VRID != n.VRID {
+				continue
```

Comment:
add the logging back?

Reply:
I don't want those logs. Anyway they are spammy and not useful.